cmd/substreams/pack.go

```diff
@@ -20,12 +24,33 @@ var packCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(packCmd)
+	packCmd.Flags().StringP("output-dir", "o", "src/pb", cli.FlagDescription(`
+		Optional flag to specify output directory to output generated .spkg file. If the received <package> argument is a local Substreams manifest file
```

Comment:
Since a default value exists, all flags are technically optional, so let's change the wording a little bit
```suggestion
		Sspecify output directory where the output generated .spkg file. If the received <package> argument is a local Substreams manifest file
```

And let's remove the thing about relative path if we decide to remove that feature from `pack`.

Reply:
Well yes, I agree they usually are. However the protogen cmd has two "mandatory flags" that need to be specified so I thought it best to distinguish between the two.

Reply:
👍